### DISCUSSION TITLE
Remove duplicate package django-stripe-payments

### DISCUSSION THREAD

**Discussion by wolph** [djangopackages/djangopackages#649] [Packages]
The package django-stripe-payments: https://djangopackages.org/packages/p/django-stripe-payments/
Has been renamed to pinax-stripe: https://djangopackages.org/packages/p/pinax-stripe/

So the old package needs to be removed. Ideally djangopackages should detect these types of duplicates (the github repo redirects) and remove them automatically.

**Comment by hashlash** [comment]
[`draceditor`](https://djangopackages.org/packages/p/draceditor/) has also been renamed to [`martor`](https://djangopackages.org/packages/p/martor/). And I think there are lots of other packages with similar case.

The auto-detection idea from @WoLpH sounds nice. Or maybe provide a way to remove a package manually (we can't do that currently can't we?). I don't know which one is better/easier tho.

**Comment by jefftriplett** [comment]
We have some backend support now to mark a package as deprecated by another package. The front end puts a banner up which should make it more obvious too. We don't have a frontend tool yet to let people mark these (at least outside of the Django Admin. 

See: https://djangopackages.org/packages/p/django-stripe-payments/

and: https://djangopackages.org/packages/p/pinax-stripe/ 

which now reference: https://djangopackages.org/packages/p/pinax-stripe-light/
